sdk/keyvault/keyvault-secrets/tests/recoverBackupRestore.test.ts

```diff
@@ -104,7 +107,10 @@ describe("Secret client - restore secrets and recover backups", () => {
     await client.setSecret(secretName, "RSA");
     const result = await client.backupSecret(secretName);
     assert.equal(Buffer.isBuffer(result), true, "Unexpected return value from backupSecret()");
-		assert.ok(result.length > 4500, `Unexpected length (${result.length}) of buffer from backupSecret()`);
+    assert.ok(
+      result.length > 4500,
+      `Unexpected length (${result.length}) of buffer from backupSecret()`
+    );
```

Comment:
it's fine, but general note - try to only do what the PR is about. I'm guessing this just slipped in

Reply:
oh this is probably VSCode auto-formatting stuff